nixos/modules/services/misc/paperless.nix

```diff
@@ -199,9 +207,41 @@ in
       defaultText = literalExpression "pkgs.paperless-ngx";
       description = lib.mdDoc "The Paperless package to use.";
     };
+
+    backup = {
+
+      enable = mkEnableOption (lib.mdDoc "Paperless backups");
+
+      directory = mkOption {
+        type = types.str;
+        default = cfg.dataDir + "/backup";
+        description = lib.mdDoc "Directory to store backups.";
+      };
+
+      calendar = mkOption {
+        type = types.str;
+        default = "01:30:00";
+        description = lib.mdDoc ''
+          Configured when to run the backup service systemd unit (DayOfWeek Year-Month-Day Hour:Minute:Second).
+        '';
+      };
+
+      options = mkOption {
+         type = types.str;
+         default = "-c -d";
```

Comment:
Please use long options by default as they explain themselves better.

I'd also prefer this to be a more abstract type. A listOf string or perhabs even an attrset via `lib.cli.toGNUCommandLineShell`.